models/dfk/dfk_hero_sales.sql

```diff
@@ -0,0 +1,54 @@
+{{ 
+     config(
+         materialized = 'incremental',
+         unique_key = 'tx_hash',
+         incremental_strategy = 'delete+insert',
+         tags = ['dfk'],
+         cluster_by = ['block_timestamp']
+     ) 
+}}
+
+with
+logs as (
+    select 
+        *
+    from {{ ref('logs') }}
+    where {{ incremental_load_filter("block_timestamp") }}
+),
+auction_created as (
+    select
+        tx_hash,
+        block_timestamp,
+        concat('0x',LTRIM(substr(topics[1], 3, 64),'0')) as seller_address,
+        java_hextoint(substr(data,3,64))::number as auction_id
+    from {{ ref('logs') }} -- reverse lookup, need to look at all the data
+    where topics[0]::string = '0x9a33d4a1b0a13cd8ff614a080df31b4b20c845e5cde181e3ae6f818f62b6ddde' -- auction created event
+),
+hero_auction_txns as (
+    select 
+        tx_hash,
+        block_timestamp,
+        java_hextoint(replace(topics[1]::string, '0x','' )) as hero_token_id,
+        java_hextoint(substr(data, 3, 64)) as auction_id,
+        java_hextoint(substr(data, 3+64, 64))::number / 1e18  as total_jewels,
+        java_hextoint(substr(data, 3+64, 64))::number / 1e18 * 0.0375 as tax_jewels,
+        concat('0x',LTRIM(substr(data, 3+128, 64),'0')) as buyer_address,
+        total_jewels * jewel_price.price  as total_usd,
+        tax_jewels * jewel_price.price as tax_usd
+    from logs
```

Comment:
recommend add alias to this and the columns above

Reply:
Not sure what this meant...

Reply:
oh such as logs.tx_hash or if you put alias `log as l` then `l.tx_hash`
as it's not clear which table the column is coming from when you join

Reply:
hmm, but this is the alias from its own table, so I'm not too sure how to alias this?
(Will have the same problem as https://github.com/MetricsDAO/harmony_dbt/pull/82/files)